yubikey/start.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
```

Comment:
I would either move files that get copied inside the container in a subdirectory, or document the situation by providing a .dockerignore -file that lists these files (this file) that are to be used on host side (only).